lib/Agrammon/Web/APIRoutes.pm6

```diff
@@ -28,14 +26,64 @@ my class AgrammonAPITokenMiddleware does Cro::APIToken::Middleware {
 }
 
 sub api-routes(Agrammon::Web::Service $ws) is export {
+    my $schema = 'share/agrammon-rest.openapi';
     route {
+        if %*ENV<AGRAMMON_DEBUG> {
+            before {
+                # Consume and re-instate request.
+                my $blob = await request.body-blob;
+                request.set-body($blob);
+                # Dump.
+                my $req = ~request;
+                try $req ~= $blob.decode('utf-8');
+                note "request=$req";
+            }
+        }
         before AgrammonAPITokenMiddleware.new(manager => get-api-token-manager());
 
-        get -> APIUser $user, 'greet' {
-            content 'application/json', { message => "Hello $user.firstname()" }
-        }
+        include rest-api-routes($schema, $ws);
     }
 }
 
+sub rest-api-routes (Str $schema, Agrammon::Web::Service $ws) is export {
+    openapi $schema.IO, {
+        operation 'greetUser', -> APIUser $user {
+            content 'application/json', { message => "Hello $user.firstname()" }
+        }
```

Comment:
Do we really want to keep this?

Reply:
I found it a cute test route, but you're right. Removed.